content/resources/radio.md

```diff
@@ -5,55 +5,71 @@ menu:
     main:
         parent: resources
 ---
+<style>
+.sstv img {
+	width: 32%;
+}
 
-# History

```

Comment:
Would it not be good to have a history section? Though I agree replacing this copy would be a good idea.